modules/ROOT/pages/authentication-authorization/dbms-administration.adoc

```diff
@@ -2001,7 +2158,8 @@ As the query result shows, access to any setting starting with `dbms.security` a
 [[access-control-dbms-administration-all]]
 == Granting `ALL DBMS PRIVILEGES`
 
-The right to perform the following privileges can be achieved with a single command:
+You can grant the `ALL DBMS PRIVILEGES` privilege to a role. +
+The `ALL DBMS PRIVILEGES` privilege is equivalent to granting the following privileges:
 
 * Create, drop, assign, remove, and show roles.
 * Create, alter, drop, show, and impersonate users.
```

Comment:
since I can't comment below:
```
* Create, alter, and drop databases and aliases.
* Enable, alter, rename, reallocate, deallocate, and drop servers
```
should both probably also say `show`? (show databases might not be a privilege as such but show aliases are at least and those lines are combined into one so (plus the database privileges affect which databases are returned by show databases so it's not that wrong 🤷))